.gitattributes

```diff
@@ -1,11 +1,8 @@
 # Path-based git attributes
 # https://www.kernel.org/pub/software/scm/git/docs/gitattributes.html
 
-# Ignore all test and documentation with "export-ignore".
-/.gitattributes     export-ignore
-/.gitignore         export-ignore
-/.travis.yml        export-ignore
+# Ignore all dot files, test and documentation with "export-ignore".
+/.*                 export-ignore
```

Comment:
`.phpstorm.meta.php` should be kept as it provides IDE code completion hints to end users